apps/publish.sh

```diff
@@ -30,13 +30,20 @@ PUSH_TARGETS=${PUSH_TARGETS-true}
 
 MACHINES=${MACHINES-""}
 PLATFORMS=${MANIFEST_PLATFORMS_DEFAULT-""}
+DONT_LOAD_CERTS=${DONT_LOAD_CERTS-""}
```

Comment:
Seems like we should make `load_extra_certs` work better/different and not introduce a new environment variable.

Reply:
I'll see what I can do.

Reply:
I am not sure it is critical.

Reply:
Its creating a bit of an "API". What's the problem you see with load_extra_certs?

Reply:
I used to get error when I was running `test_apps_publishing.sh`.
Somehow, I am getting only warning now, so I am removing this change.

```
== 2024-02-29 15:09:01 Loading extra ca certificates
== 2024-02-29 15:09:01 Loading extra ca certificates
WARNING: ca-certificates.crt does not contain exactly one certificate or CRL: skipping
```

Reply:
I removed the commit.